ONSdigital/eq-survey-runner
# eq-43 Landing Page

<img width="901" alt="screen shot 2016-04-06 at 08 43 50" src="https://cloud.githubusercontent.com/assets/930398/14309326/baf281d8-fbd3-11e5-9794-eeb79499d576.png">
## Changes
- re-named "cover page" to "landing page" in system
- removed `_cover.scss` and implemented layout with just utility classes
- added `accordion` component with associated template, styles, scripting and tests
- added `notice` component with styles
## Test
- `npm run compile`
- `npm run test`
- visit landing page of survey, confirm it looks like the visual above
- click "How is you information used?" to test accordion works
